src/resize.cc

```diff
@@ -840,6 +869,17 @@ class ResizeWorker : public NanAsyncWorker {
       image = composited;
     }
 
+    // Reverse premultiplication after all transformations:
+    if (shouldPremultiplyAlpha) {
+      VipsImage *imageUnpremultiplied;
+      if (Unpremultiply(hook, image, &imageUnpremultiplied)) {
+        (baton->err).append("Failed to unpremultiply alpha channel.");
+        return Error();
+      }
+
```

Comment:
Thanks! @yoojin53 taught me how to use Valgrind and I found this and another leak: da24175a0cd3ae258163db0db6c26b94a4ba953e

We still found quite a few reports about `ResizeWorkerExecute` but none of them matched a missing `vips_object_local`.